MapboxVisionAR/Render/ARScene/ConcreteNodes/ARLaneNode/ARLaneNode.swift

```diff
@@ -0,0 +1,107 @@
+import simd
+
+/// Represents AR Lane that can be rendered as a part of `ARScene`
+class ARLaneNode: ARNode {
+    // MARK: - Properties
+
+    /// Type of node. Always returns `rootNode`.
+    private(set) var nodeType: ARNodeType
+    /// Underlying AR entity.
+    private(set) var entity: AREntity?
```

Comment:
You have class `ARLaneEntity` but use `AREntity` here, seems odd

Reply:
To be honest I'm not sure we need to have `AREntity` at all, but the current implementation is a kind of mirroring of `ARRenderer.swift`. I believe I can perform additional refactoring as a part of separate PR. Does that make sense?

Reply:
You're referring to the "current implementation", but this implementation was made in this pr. If it is a part of separate PR it shouldn't be in this pr

Reply:
In other words you're saying that I shouldn't review this code, it means I should approve unreviewed code, means `dev` will contain unreviewed code, I think it's not the right way. Am I wrong?

Reply:
I didn't say "do not review the code". I'm just trying to say: "Please review code but focus on initial task".
Please see comment above https://github.com/mapbox/mapbox-vision-ios/pull/118#discussion_r292799926.

Do you agree?